lgn-messages/src/types/v1/preprocessing/ext_tasks.rs

```diff
@@ -265,48 +260,29 @@ impl FinalExtraction {
 
     pub fn new_merge_table(
         table_id: TableId,
+        simple_table_hash: TableHash,
+        mapping_table_hash: TableHash,
         block_nr: BlockNr,
         contract: Address,
-        mapping_table_hash: u64,
-        simple_table_hash: u64,
-        table_value_proof_version: MptNodeVersion,
+        value_proof_version: MptNodeVersion,
```

Comment:
The reason there is only one value proof version is because it's the same block and same root hash of the storage trie of the contract .
I'm writing it just as info because it took me some time to make sure it was correct, maybe worth putting a comment there.

Reply:
That's actually a requirement.

Reply:
added a comment to the struct